prompt.py

```diff
@@ -959,7 +958,7 @@ def check_privatenet():
         nonce_chain = open(neopy_chain_meta_filename, "r").read()
         if nonce_chain != nonce_container:
             raise PrivnetWrongChainDatabaseError(
-                "Chain database in Chains/privnet is for a different private network than the current container. "
+                "Chain database in Chains/privnet is for a different private network than the current container (or RPC is not enabled). "
```

Comment:
If RPC isn't running, the code should short out above, right? So, I don't think this exception will be raised when RPC isn't running...